src/content/es/2020/http2.md

```diff
@@ -0,0 +1,690 @@
+  sheets_gid="152400778",
+  sql_file="percentiles_of_resources_loaded_over_HTTP2_or_better_per_site.sql"
+  )
+}}
+
+La Figura 22.6 compara cuánto HTTP/2 o gQUIC se usa en un sitio web entre este año y el año pasado. El cambio más notable es que más de la mitad de los sitios ahora tienen el 75% o más de sus solicitudes atendidas a través de HTTP/2 o gQUIC en comparación con el 46% del año pasado. Menos del 7% de los sitios no realizan solicitudes HTTP/2 o gQUIC, mientras que (solo) el 10% de los sitios son solicitudes HTTP/2 o gQUIC en su totalidad.
+
+¿Qué pasa con el desglose de la página en sí? Por lo general, hablamos de la diferencia entre contenido propio y de terceros. Tercero se define como contenido que no está bajo el control directo del propietario del sitio, que proporciona funciones como publicidad, marketing o análisis. La definición de terceros conocidos se toma del repositorio <a href="https://github.com/patrickhulce/third-party-web/blob/8afa2d8cadddec8f0db39e7d715c07e85fb0f8ec/data/entities.json5" hreflang="en">third party web</a>.
+
+La Figura 22.7 ordena cada sitio web por la fracción de solicitudes HTTP/2 para terceros conocidos o solicitudes de contenido propio en comparación con otras solicitudes. Existe una diferencia notable, ya que más del 40% de todos los sitios no tienen del todo solicitudes HTTP/2 o gQUIC. Por el contrario, incluso el más bajo 5% de las páginas tiene un 30% de contenido de terceros servido a través de HTTP/2. Esto indica que una gran parte de la amplia adopción de HTTP/2 es impulsada por terceros.
+
+{{ figure_markup(
+  image="http2-first-and-third-party-http2-usage.png",
+  caption="Distribución de la fracción de solicitudes HTTP/2 de terceros y de contenido propio por página.",
+  description="Un gráfico de líneas que compara la fracción de solicitudes HTTP/2 de contenido propio con solicitudes HTTP/2 o gQUIC de terceros. El gráfico ordena los sitios web por fracción de solicitudes HTTP/2. El 45% de los sitios web no tienen solicitudes HTTP/2 de contenido propio. Más de la mitad de los sitios web atienden solicitudes de terceros solo a través de HTTP/2 o gQUIC. El 80% de los sitios web tienen 76% o más solicitudes HTTP/2 o gQUIC de terceros.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSOkWXtrbMfTLdhlKbBGDjRU3zKbnCQi3iPhfuKaFs5mj4smEzInDCYEnk63gBdgsJ3GFk2gf4FOKCU/pubchart?oid=1409316276&format=interactive",
+  sql_file="http2_1st_party_vs_3rd_party.sql",
+  sheets_gid="733872185"
+)
+}}
+¿Hay alguna diferencia ente cuáles tipos de contenido se sirven a través de HTTP/2 o gQUIC? La Figura 22.8 muestra, por ejemplo, que el 90% de los sitios web sirven el 100% de fuentes y audio de terceros a través de HTTP/2 o gQUIC, solo el 5% a través de HTTP/1.1 y un 5% son una combinación. La mayoría de los activos de terceros son scripts o imágenes, y se sirven únicamente a través de HTTP/2 o gQUIC en el 60% y el 70% de los sitios web, respectivamente.
```

Comment:
We like to keep line numbering consistent with English translation to make it easier to find the same lines for non-Spanish speakers (though looks like in a few places we'll need to fix the English version too - if you could add to this commit that would be great!)

```suggestion

¿Hay alguna diferencia ente cuáles tipos de contenido se sirven a través de HTTP/2 o gQUIC? La Figura 22.8 muestra, por ejemplo, que el 90% de los sitios web sirven el 100% de fuentes y audio de terceros a través de HTTP/2 o gQUIC, solo el 5% a través de HTTP/1.1 y un 5% son una combinación. La mayoría de los activos de terceros son scripts o imágenes, y se sirven únicamente a través de HTTP/2 o gQUIC en el 60% y el 70% de los sitios web, respectivamente.
```